### ISSUE TITLE
HamburgerMenu.HamburgerBackground Overrides Containing Page Background

### ISSUE THREAD

**Issue by MarkDHarris** [Windows-XAML/Template10#722]
Instead of NuGet I am using Template10 as a submodule and referencing the (Library) and (Service) projects directly in my project to learn more about how they work.  

1)  I am overriding the OnInitializeAsync function of App.xaml.cs and using this code to set a background image in the shell so that it applies to every page.

``` csharp
Window.Current.Content = new Views.Shell((NavigationService)nav)
   {
         Background = new ImageBrush()
         {
              ImageSource = new BitmapImage(new Uri("ms-appx:///Assets/Background/Background-Landscape-2560x1440.png", UriKind.Absolute)),
              Stretch = Stretch.UniformToFill,
              Opacity = 1.0,
              AlignmentX = AlignmentX.Center,
              AlignmentY = AlignmentY.Center
        }
   };
```

2)  I am using this XAML in the shell page to control the hamburger menu properties.

``` xaml
<Controls:HamburgerMenu x:Name="MyHamburgerMenu" 
             IsFullScreen="False" 
             HamburgerBackground="Green"
             NavAreaBackground="Green"                                     
             VisualStateNarrowMinWidth="-1" 
             VisualStateNormalMinWidth="0"
      VisualStateWideMinWidth="-1" >
```

3)  I was working with the commit hash ending in c3582c8 and dated 2/19/2016 at 4:57:19 PM and everything was good – background image displayed on all the pages and the hamburger background was green.  I then pulled the latest this morning – commit hash ending 755de dated 3/2/2016 at 1:08:18 AM (all times EST I think) and I immediately noticed that all my pages (and the entire area of the page) displayed green and no longer displayed the background image.  I reverted to the previous commit and it displayed correctly again.  I tracked it down to the HamburgerBackground setting.  If I change that from Green to Transparent, my shell background image displays correctly (but obviously the HamburgerButton is now transparent which is undesirable).  

It seems like the HamburgerBackground setting is overriding the entire page’s and shell’s background settings and not just the Hamburger button itself.  

Is this a bug or my misunderstanding of how this should work?

**Comment by JerryNixon** [comment]
Could you confirm that your page (or your root element) has a background color? And is not transparent?

**Comment by MarkDHarris** [comment]
I didn’t have a background color set in Shell.xaml, instead I was setting the background property to an image in the code I showed with my question.  

I did a little more experimentation:

I commented out the code that sets the shell background to my image and instead set the Background property to orange in the xaml.  That had no effect (it stayed Green which is what I had the HamburgerBackground property set to be as you can see in my first post).  I then set the HambugerBackground to transparent and that resulted in the HamburgerButton and every page contained by the shell to then be orange.  (The individual page’s backgrounds seem to be overridden by the shell's properties – I’m not sure if that is intended or not but I was actually relying on that so I could set the background image once in the shell and have every page use that image.)  

When I changed the HamburgerBackground back to a different color instead of transparent it again overrode everything and the HamburgerButton and every page contained by the shell was that new color.

So, the dominating property is definitely the HamburgerBackground property but I don’t understand why and so far I can only narrow it down to a change within the last 12 days.

**Comment by angusbreno** [comment]
Duplicated #699

**Comment by JerryNixon** [comment]
Fixed in 1.1.10. If you see anything else, please open a new issue. Thanks.
